src/sidebar/store/modules/selection.js

```diff
@@ -92,8 +92,8 @@ const setTab = (selectedTab, newTab) => {
 
 function init(settings) {
   return {
-    // Contains a map of annotation tag:true pairs.
-    focusedAnnotationMap: null,
+    // An array of annotation `$tag`s representing annotations that are focused
+    focusedAnnotations: [],
```

Comment:
This isn't a map, so it has been renamed.

Reply:
Come to think of it, is it ever possible to focus more than one annotation at a time. Could this just be the tag of the focused annotation instead?

Reply:
It is indeed—when on the host page, if one hovers over text with more than one annotation associated, all will be focused.

Reply:
Now that I think more about it, I'm not sure we even need to store the focused annotations in the store at all. From a quick skim it looks like the only components that use it are `ThreadCard` some descendants. It could potentially just be handled within those components. The advantage is that local state updates are cheaper than store ones.

Reply:
Please ignore the above. @lyzadanger reminded me that this is in the store because the focused annotations can be set from the annotator side of the app as well.